src/output/outputMod.F90

```diff
@@ -1393,6 +1393,7 @@ subroutine storeSurfsolInBuffer(sps, buffer, nn, blockID,   &
     use utils, only : setPointers
     use flowUtils, only : computePtot
     use inputCostFunctions
+    use cgnsGrid, only : cgnsDoms,cgnsNDom ! see l. 4105 in preprocessingAPI.F90
```

Comment:
This should probably just refer to `subroutine updateRotationRate` in the mentioned file. I guess the use is self-explanatory, given the function name?